### PR TITLE
Updated prettier config files

### PR DESCRIPTION
**Pull request by knowncitizen**
- rewrote prettier initiation in package.json
 - added .prettierignore for vendor files (and anything else)
 - added proseWrap: always to prettierrc
 - committed changes from a prettier run

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "prettier": "prettier --write 'src/**/*.{js,jsx,ts,tsx}'",
+    "prettier": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md,scss,yaml,yml}\"",
```
Comment (jtomasek) [inline package.json]: By keeping the single quotes you could avoid having to escape them.

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1 @@
+vendor/
```
Comment (jtomasek) [inline .prettierignore]: `build/` should be also ignored IMHO

Comment (knowncitizen) [reply .prettierignore]: Done

File: package.json (hunk 1 above)
Comment (knowncitizen) [reply package.json]: Done

**Pull request merged**